### PR TITLE
Add day of week

### PR DESCRIPTION
**Pull request by tikismoke**
Need to ALTER TABLE EVENT to add a newcolumn dayofweek.

### CODE REVIEW

File: plugins/eventManager/eventmanager.plugin.disabled.php [hunk 1]
```diff
@@ -145,24 +147,45 @@ function eventmanager_plugin_page($_){
...
+
 								    	<?php } ?>
 								    </select>
 								</div>
-
 								<div class="span2">
 									<label for="eventMonth">Mois</label>
 								    <select class="input-medium" name="eventMonth" id="eventMonth">
 								    		<option <?php  if($currentEvent->getMonth()=='*') echo 'selected="selected"';  ?> value="*">Tous</option>
-								    	<?php for($i=1;$i<13;$i++){ ?>
```
Comment (ldleman) [inline plugins/eventManager/eventmanager.plugin.disabled.php]: Why ?

**Comment by tikismoke** [comment]
Don't remember :(  This was month ago.

If I remember the value are stored in dB but that didn't work without altering it.

**Comment by ldleman** [comment]
Damn it! Already 1 month ? I've some brain lags :D !
Everything seem to be fine but that part is nasty, could you update it to keep loop before merge ?

**Comment by tikismoke** [comment]
Sorry idle I will not have no time to that before October ;) lot's of other projects and family to look at before ;)

**Comment by ldleman** [comment]
No problems :), don't hesitate to repull if u have time, good luck for ure other projects (and family obviously) :)

**Pull request closed without merging**